src/applications/vaos/appointment-list/index.jsx

```diff
@@ -1,40 +1,19 @@
 import React from 'react';
-import { useSelector } from 'react-redux';
 import { Switch, Route } from 'react-router-dom';
-import { selectFeatureHomepageRefresh } from '../redux/selectors';
-import PageLayout from './components/AppointmentsPage/PageLayout';
 import AppointmentsPageV2 from './components/AppointmentsPageV2/index';
-import AppointmentsPage from './components/AppointmentsPage/index';
 import RequestedAppointmentDetailsPage from './components/RequestedAppointmentDetailsPage';
 import ConfirmedAppointmentDetailsPage from './components/ConfirmedAppointmentDetailsPage';
 import CommunityCareAppointmentDetailsPage from './components/CommunityCareAppointmentDetailsPage';
 import useManualScrollRestoration from '../hooks/useManualScrollRestoration';
 
 function AppointmentListSection() {
-  const featureHomepageRefresh = useSelector(state =>
-    selectFeatureHomepageRefresh(state),
-  );
   useManualScrollRestoration();
   return (
     <Switch>
       <Route path="/cc/:id" component={CommunityCareAppointmentDetailsPage} />
       <Route path="/va/:id" component={ConfirmedAppointmentDetailsPage} />
       <Route path="/requests/:id" component={RequestedAppointmentDetailsPage} />
-      <Route
-        path="/"
-        render={() => {
-          let content = <AppointmentsPage />;
-          if (featureHomepageRefresh) {
-            content = <AppointmentsPageV2 />;
-          }
-
-          return (
-            <PageLayout showBreadcrumbs showNeedHelp>
-              {content}
-            </PageLayout>
-          );
-        }}
-      />
+      <Route path="/" component={AppointmentsPageV2} />
```

Comment:
Do we want to keep the `v2` naming convention for these components now that they're the default?

Reply:
I think that's a good change, but it looks like it blows up the diff and is a little risky, so I think we should do it in separate PR.